File: 2828. Check if a String Is an Acronym of Words/code.cpp
#include <iostream>
#include <string>

using namespace std;

bool isAcronym(string words[], int size, const string& s) {
    string acronym = "";
    for (int i = 0; i < size; i++) {
        acronym += words[i][0];
    }
    return acronym == s;
}

int main() {
    string words[] = {"alice", "bob", "charlie"}; 
    int size = sizeof(words) / sizeof(words[0]);
    string s = "abc";
    bool result = isAcronym(words, size, s);
    cout << (result ? "true" : "false") << endl;
    return 0;
}

File: 2828. Check if a String Is an Acronym of Words/code.py
def isAcronym(words, s) -> bool:
    # Step 1: Extract the first character of each word
    # acronym = "".join(word[0] for word in words)
    acronym = ""
    for word in words:
        acronym +=word[0]
    
    # Step 2: Compare the concatenated string with s
    print(acronym)
    return acronym == s


words = ["alice","bob","charlie"]
s = "abc"
ans = isAcronym(words,s)
print(ans)